package.json

```diff
@@ -141,7 +141,7 @@
     "fabric": "^4.1.0",
     "firebase": "^8.0.0",
     "firebase-tools": "^9.1.0",
-    "firebaseui": "4.7.1",
+    "firebaseui": "https://github.com/oppia/firebaseui-web/releases/download/v4.7.1-PATCH/firebaseui-4.7.1-PATCH.tgz",
```

Comment:
Why are we using our fork? Also would it be possible to simplify the address similar to `htmllint-cli` in this file?

Reply:
We need this fork because of: firebase/firebaseui-web#778. Due to that issue, the `firebaseui` package ignores emulator configurations, even when it's set-up correctly (like in this PR). Our fork adds a hacky fix to this bug (inspired by https://github.com/firebase/firebaseui-web/issues/778#issuecomment-744159663), hence why we need to use it.

---

I tried setting it up as you described, but it seems like yarn really needs to point to a `.tar.gz` file to setup the package correctly. I can't get it to work without specifying the file, I get errors like on the frontend:
```
Uncaught SyntaxError: Unexpected token '!'
    at Object../node_modules/firebaseui-angular/fesm2015/firebaseui-angular.js (vendors~about~admin~classroom~collection_editor~collection_player~console_errors~contact~contributor~0f69d550.bundle.js:2103)
    at __webpack_require__ (runtime.bundle.js:80)
    at eval (shared-component.module.ts:13)
    at Module../core/templates/components/shared-component.module.ts (about~admin~classroom~collection_editor~collection_player~console_errors~contact~contributor_dashboa~9bda85c1.bundle.js:589)
    at __webpack_require__ (runtime.bundle.js:80)
    at eval (splash-page.module.ts:8)
    at Module../core/templates/pages/splash-page/splash-page.module.ts (splash.bundle.js:46)
    at __webpack_require__ (runtime.bundle.js:80)
    at eval (splash-page.import.ts:31)
    at Module../core/templates/pages/splash-page/splash-page.import.ts (splash.bundle.js:34)
```

Reply:
Okay, thanks for the explanation, can you please follow firebase/firebaseui-web#778, so that we know when it is fixed and start using regular version?

Reply:
Done: #11802